ui/tests/components/table/table-element/grouping-path-cell.test.tsx

```diff
@@ -0,0 +1,68 @@
+import { fireEvent, render, screen, waitFor } from '@testing-library/react';
+import GroupingPathCell from '@/components/table/table-element/grouping-path-cell';
+import userEvent from '@testing-library/user-event';
+
+beforeEach(() => {
+    if (!navigator.clipboard) {
+        Object.defineProperty(navigator, 'clipboard', {
+            value: {
+                writeText: jest.fn(() => Promise.resolve())
+            },
+            writable: true
+        });
+    } else {
+        jest.spyOn(navigator.clipboard, 'writeText').mockImplementation(() => Promise.resolve());
+    }
+});
```

Comment:
Question, why is the if, else block needed? I would assume having both Object.defineProperty and jest.spyOn would be fine running together.

Reply:
to check if the navigator.clipboard  is available/exist

Reply:
done